scc/dockerfile-based-builds.hbs.md

```diff
@@ -47,6 +47,7 @@ Dockerfile by passing the `dockerfile` parameter:
 $ tanzu apps workload create foo \
```

Comment:
This should ideally be prefixed with something different like `my-` so that way it's a different value that `part-of`.